subsys/logging/log_msg.c

```diff
@@ -150,6 +150,7 @@ static void msg_free(struct log_msg *msg)
 			log_free((void *)(str));
 		}
 	} else {
+		; /* Non-empty final else in if ... else if struct */
```

Comment:
`Message does not contain any arguments that might be a transient string, No action required.`